tests/test_producers.py

```diff
@@ -79,8 +105,8 @@ def test_message_enqueued(self):
 
 class CeleryProducerTest(TestCase):
     @classmethod
-    def setUpClass(cls):
-        super(CeleryProducerTest, cls).setUpClass()
+    def setUpTestData(cls):
+        super(CeleryProducerTest, cls).setUpTestData()
```

Comment:
i believe you don't need to call super in setUpTestData, as it's a hook

Reply:
oh. true